lib/CleantalkSP/SpbctWP/State.php

```diff
@@ -123,6 +123,7 @@ class State extends \CleantalkSP\Common\State
 		'notice_show'              => null,
 		'notice_renew'             => false,
 		'notice_trial'             => false,
+        'notice_review'            => false,
```

Comment:
Please, align it in one column with other properties.

Reply:
I have spaces, not tabs, which meets the requirements of Psalm